ice40/devices/top-routing-virt/CMakeLists.txt

```diff
@@ -1,4 +1,44 @@
 add_subdirectory(tiles)
 
 add_file_target(FILE arch.xml SCANNER_TYPE xml)
-define_device_type(DEVICE_TYPE top-routing-virt ARCH ice40 ARCH_XML arch.xml)
+
+get_target_property_required(CONDA_DIR env CONDA_DIR)
+get_target_property_required(PYTHON3 env PYTHON3)
+
+set(ICEBOX_TIMING ${symbiflow-arch-defs_SOURCE_DIR}/third_party/icestorm/icefuzz/timings.py)
+set(CLEAN_ICEBOX_TIMING cleaned_timing.txt)
+set(TIMING_TXT_FILE ${CONDA_DIR}/share/icebox/timings_hx1k.txt)
```

Comment:
Rather than using CONDA_DIR directly, I recommend adding an icebox directory CACHE variable that defaults to ${CONDA_DIR}/share/icebox/timings_hx1k.txt, etc.